src/components/Select/Select.stories.tsx

```diff
@@ -405,6 +407,44 @@ export const StyledUncontrolled: StoryObj = {
   },
 };
 
+/**
+ * You can select multiple values by passing `multiple` to the parent element. When doing this,
+ * make sure all props that use the value (e.g., `value` and `defaultValue`) should use an array instead
+ * of an object or value for the individual `Select.Option` entries.
+ *
+ * When handling the button text, `value` represents the data for all options selected. This allows for a flexible
+ * layout to fit the needs of the design.
```

Comment:
TODO: add documentation and/or another story showing what fields get generated when multiple values are selected.